service/version_bundles.go

```diff
@@ -158,6 +158,12 @@ func NewVersionBundles() []versionbundle.Bundle {
 		},
 		{
 			Changelogs: []versionbundle.Changelog{
+				{
+
+					Component:   "Container linux",
```

Comment:
LoL @calvix it's `"containerlinux"` <3

Reply:
docs says its Container linux with the space
https://coreos.com/os/docs/latest/

Reply:
kubectl also report "Container linux"

Reply:
I understand. This is how team agreed to call and display this name. Disclaimer: It wasn't my idea.

Reply:
lol, what a fine decision, we can call it `shubgregute` too since it doesn't make sense :P  (joking not directed at you)